src/i18n/index.js

```diff
@@ -1,3 +1,7 @@
+import { messages as paragonMessages } from '@edx/paragon';
+import { messages as headerMessages } from '@edx/frontend-component-header';
+import { messages as footerMessages } from '@edx/frontend-component-footer';
```

Comment:
update these redirects as well

Reply:
I couldn't install `@openedx/paragon`, it seems that it's either private in `npm` or I'm not understanding how redirects works.

Reply:
Ahhh I thought this was a GitHub path (https://github.com/edx/paragon redirects to https://github.com/openedx/paragon) but if this is `npm` I am out of touch and probably completely wrong. Sorry about that! 😅